Formula/d/decasify.rb

```diff
@@ -0,0 +1,33 @@
+class Decasify < Formula
+  desc "Utility for casting strings to title-case according to locale-aware style guides"
+  homepage "https://github.com/alerque/decasify"
+  url "https://github.com/alerque/decasify/releases/download/v0.7.6/decasify-0.7.6.tar.zst"
+  sha256 "299e462cf5a93b5a7da865764ca94729111945349f5fb6e2e79b5bbeff574cd9"
+  license "LGPL-3.0-only"
+
+  head do
+    url "https://github.com/alerque/decasify.git", branch: "master"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
+  depends_on "jq" => :build
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+
+  uses_from_macos "zlib"
+
+  def install
+    system "./bootstrap.sh" if build.head?
+    system "./configure", *std_configure_args, "--disable-silent-rules"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    assert_match "decasify v#{version.to_s.match(/\d\.\d\.\d/)}", shell_output("#{bin}/decasify --version")
```

Comment:
```suggestion
    assert_match "decasify v#{version}", shell_output("#{bin}/decasify --version")
```
Don't think you need the rest of that.

Reply:
I *think* what that is doing is allowing the version field to match even when build from `--HEAD` which will have some other segments involved, and this keeps the dot separators from being wildcards.

In any case it originates from this commit: 51513faa41ea. So if that doesn't make any sense we should clean it up there too. Note in both cases the `--HEAD` builds will have versions *starting* with a 3 segment semver, but after that may have other variously formatted segments.

Reply:
Also (only) found it glm and osm2pgsql. Ya I have no idea what the purpose is there. Since this match isn't bounded to EOL if `#{version}` happens to already have a parsed semver ready to go, why are we shaking escapes at it?